### PR TITLE
Add Pension EP codes

### PR DESCRIPTION
**Pull request by leikkisa**
connects #7984 

Adds new EP codes if a Benefit Type is "Pension"

### CODE REVIEW

File: spec/models/claim_review_spec.rb [hunk 1]
```diff
@@ -208,6 +210,34 @@ def random_ref_id
...
+        let(:benefit_type) { "pension" }
+
+        it "creates issues and assigns pension end product codes to them" do
+          subject
+
+          expect(rating_request_issue.reload.end_product_establishment).to have_attributes(code: "030HLRRPMC")
+          expect(non_rating_request_issue.reload.end_product_establishment).to have_attributes(code: "030HLRNRPMC")
+        end
+
+        context "for a supplemental claim" do
```
Comment (kencheeto) [inline spec/models/claim_review_spec.rb]: this is a little surprising in the claim_review_spec - i would expect tests here to be covering only generic claim_review functionality. Is there a better place to put this test?

File: app/models/supplemental_claim.rb [hunk 2]
```diff
@@ -15,27 +19,10 @@ def ui_hash
...
-  end
-
-  def end_product_base_modifier
-    valid_modifiers.first
-  end
-
-  def valid_modifiers
-    END_PRODUCT_MODIFIERS
-  end
-
```
Comment (kencheeto) [inline app/models/supplemental_claim.rb]: 😍 love this cleanup

File: spec/models/higher_level_review_spec.rb [hunk 3]
```diff
@@ -269,6 +273,21 @@
...
+        let(:benefit_type) { "pension" }
+
+        it "creates end product establishment with pension ep code" do
+          subject
+
+          first_dta_request_issue = RequestIssue.find_by(rating_issue_reference_id: "rating1")
+          second_dta_request_issue = RequestIssue.find_by(rating_issue_reference_id: "rating2")
+
+          expect(first_dta_request_issue.end_product_establishment.code).to eq("040HDERPMC")
+          expect(second_dta_request_issue.end_product_establishment.code).to eq("040HDERPMC")
```
Comment (kencheeto) [inline spec/models/higher_level_review_spec.rb]: is it important to test two dta request issues?

Comment (leikkisa) [reply spec/models/higher_level_review_spec.rb]: Nope, I was just following the test above, but will simplify this.

File: spec/models/claim_review_spec.rb (hunk 1 above)
Comment (leikkisa) [reply spec/models/claim_review_spec.rb]: Good call, I can add specific tests to HLR and SC.

**Pull request merged**